plugin.json

```diff
@@ -6,7 +6,7 @@
     "license": "GPL v3+",
     "homepage": "http://piwik.org",
     "require": {
-        "piwik": ">=2.8.0-b3",
+        "piwik": ">=2.15.0",
```

Comment:
I require now Piwik 2.15.0 as this is the version I tested it with. Just to make sure it looks ok